pkg/project/v2/project_loader.go

```diff
@@ -189,6 +190,19 @@ func loadProject(fs afero.Fs, context ProjectLoaderContext, projectDefinition ma
 			errors = append(errors, newDuplicateConfigIdentifierError(c))
 		}
 	}
+
+	// The scope parameter of a key user actions web configuration needs to be a reference to another application-web config
+	// The reference parameter makes sure that rely on the fact that kua web configs are loaded/deployed within the same
+	// sub graph (independent component) later on as
+	for _, c := range configs {
```

Comment:
first i wanted to put that check into Typedefinition::Validate(), but turns out that this code is also used for deletion use case, where scope is just a string which is ok there